src/pathfora.js

```diff
@@ -866,6 +895,53 @@
       return valid;
     },
 
+    entityFieldChecker: function (widget, fieldName, found) {
+      if (!found || !found.length) {
+        return true;
+      }
+
+      // for each template found...
+      for (var f = 0; f < found.length; f++) {
+        // parse the field name
+        var dataval = found[f].slice(2).slice(0, -2),
+            parts = dataval.split('|'),
+            def;
+
+        // get the default (fallback) value
+        if (parts.length > 1) {
+          def = parts[1].trim();
+        }
+
+        // check for subfields if the value is an object
+        var split = parts[0].trim().split('.');
```

Comment:
Well, right now it's only doing this split on the fieldname, so it doesn't even look for dots in the fallback, but I see your point.